app/src/main/cpp/skyline/gpu/interconnect/maxwell_3d/pipeline_manager.cpp

```diff
@@ -942,12 +943,20 @@ namespace skyline::gpu::interconnect::maxwell3d {
         });
     }
 
-    PipelineManager::PipelineManager(GPU &gpu) {
+    PipelineManager::PipelineManager(GPU &gpu, JvmManager &jvm) {
         if (!gpu.graphicsPipelineCacheManager)
             return;
 
-        std::ifstream stream{gpu.graphicsPipelineCacheManager->OpenReadStream()};
+        u32 pipelineCount{};
+        jint compiledCount{};
+        std::ifstream stream{gpu.graphicsPipelineCacheManager->OpenReadStream(&pipelineCount)};
```

Comment:
Have this return an std pair, and then use structured binding as approved. Also name like totalPipelineCount or smth